src/Header/AcceptHeaderItem.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/**
+ * Spiral Framework.
+ *
+ * @license   MIT
+ * @author    Pavel Z
+ */
+
+declare(strict_types=1);
+
+namespace Spiral\Http\Header;
+
+/**
+ * Represents "Accept" header item.
+ */
+class AcceptHeaderItem
+{
+    /** @var string */
+    private $mime = null;
```

Comment:
No need to use `null` defaults for `$mime`, `$type` and `$subype`. It is applied by the PHP itself
```suggestion
    private $mime;
```